### ISSUE TITLE
Meklab Dropdown list appears broken.

### ISSUE THREAD

**Issue by Koniving** [MegaMek/megamek#170]
![this](https://cloud.githubusercontent.com/assets/18667806/14798944/ed0c84b4-0b07-11e6-889b-eb0ab25d523a.jpg)

I have this error with Meklab, both development and stable, as well as the one that came with Megamek HQ.
This is even after doing a completely fresh folder.
Since I'm not seeing this bug mentioned anywhere else, I'm left to assume it could be just on my end, to which I have to ask:  How do I fix?

**Comment by arlith** [comment]
It's not clear what question you're asking.  Which drop down is broken?

**Comment by HammerGS** [comment]
@Koniving I think the problem is the Estevez are a level 4 unit and you have selected to show IS Introductory units.

CTRL-Click all the fields in type drop down (the one you have in blue on your screen shot).  Once they are all selected the Estevez should show up.

Although it shouldn't matter you are also in the Mek section looking for a tank.  Its supposed to work regardless, but I would use FILE--->SWITCH UNIT TYPE-->COMBAT VEHICLE first.

**Comment by Koniving** [comment]
Fair enough on the estevez.

But I am referring to the open dropdown menu that I have not clicked.... Non existent ability to select the type (mech, tank, dropships, infantry, etc.) And I have this problem in all forms of this unit selector.  Lab, HQ, megamek.

The weightclass is not selected and its showing me a dropdown list of unit sources.

**Comment by Koniving** [comment]
None of which I have opened or clicked.

**Comment by Koniving** [comment]
I have the latest megamek HQ development release installed but tried separate folders of lab with stable and development to get the same problem.  That dropdown is always open preventing me from accessing type where it belongs.

**Comment by Koniving** [comment]
Also the latest listed stable release of HQ gave me the same problem even though placed in its own separate folder on a different hard drive.

**Comment by Dylan-M** [comment]
There is nothing that I see wrong in your screenshot, just in your understanding of the controls on the form. The "Type" is not a dropdown (combobox). It is now a listbox. That means it operates slightly different, but makes it easy to multi-select the types (sources). Speaking of which, Type is actually the list of sources. Unit Type at the top is to select the type of unit. Everything is there, and working intended. I've verified this on all possible variants of the release for Linux and OSX.

**Comment by Dylan-M** [comment]
Also, (and doing the same thing on purpose to illustrate how annoying it is) if you have a lot to say, say it rather than comment spamming 4 comments in the space of 6 minutes please. Those of us who get email notifications find it annoying. Those of us who get email AND development chat system notifications find it **very** annoying.

**Comment by pheonixstorm** [comment]
Or edit your last post to include whatever additional whatever you want to say

**Comment by Koniving** [comment]
From what I understood the ticket was closed.  I did not realize it was changed from a dropdown to a list menu and for some reason compared to a version I had from last year I thought this list menu (as opposed to the broken drop menu I thought it was) could be covering one of the selectors I'm used to using.  

Turns out it's not.  I checked the notes to see if it was mentioned but it evidently was put in before the current stable so it was unmentioned in more recent notes.

I can understand the merits of it being a list menu rather than a dropdown but it just looks so weird being dead center like that with dropdowns appearing over it.  

I would like to suggest having it be at the top, then the drop down menus below it?  Part of why I thought it was a bug is sometimes the list will highlight while selecting things from the two dropdown above.
